### DISCUSSION TITLE
SO2.4.10 "No Sysmon Logs"

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#11056] [2.4] [answered]
I am currently running Security Onion 2.4.10 in standalone mode. Upon installing the elastic agent on a couple of hosts, I cannot see any Sysmon logs. Any suggestion? Everything is in working order from what I can tell. Sysmon is running on the endpoints and the settings are enabled in the initial endpoints policy to capture Sysmon events. I even added a custom event log integration package to capture Security logs which works perfectly. Perhaps something wrong with the initial endpoints package??
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/a769ac27-83ac-449b-ab0a-d2779ced36ec)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/4b4eb61b-e1ec-4cb4-ae58-8a8d38d8aaa0)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/c989ae7c-18cc-4794-a211-d00a2c748737)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/66588bfa-d562-4047-be9c-dfff77449951)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/37a19c48-7c1b-4130-bcc0-62426b807e8f)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/08528982-4655-4f0a-9073-eea9e9d54ae9)

**Comment by weslambert** [comment]
A couple of questions:

- If you navigate to the Windows integration and go to `Settings`, what version does it show?
- If you navigate to Kibana -> Stack Management -> Index Management -> Component Templates -> search for sysmon, and view the package template, then navigate to settings what does it show for the default pipeline value?

**Reply by ColeVan** [reply]
Thanks for getting back to me. Here are the screenshots to your questions.
Windows integration
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/549d94f9-01d3-4452-ab9b-49cf0398b10a)

Sysmon templates. There seem to be two. One of the component templates has nothing in it.

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/3a82ceea-39fb-4829-b4bf-2c65cfd66a52)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/813a89c9-3052-4d69-b650-66eaa1029640)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/2a7c573a-84fa-44a9-9d71-982164a81127)

**Reply by weslambert** [reply]
Try clicking `Reinstall Windows` from the Windows integration. After that, let me know if the default pipeline value in the `@package` template changes , matches the version of the Windows integration, and if you receive Sysmon logs.

**Comment by ColeVan** [comment]
Seems to have updated the template to 1.24. I'm seeing sysmon logs now.
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/06c88a1f-afa6-44b3-894c-2620e8cb54c4)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/1057d0cb-d9ac-473d-9f94-31bb17d5bb97)

**Comment by 0nnyx** [comment]
Faced the same issue on a fresh install of 2.4.10 standalone. Got resolved after doing Reinstall Windows from the Windows integration.
Pretty sure other people are also facing this issue but didn't notice it yet.

**Comment by TOoSmOotH** [comment, marked as answer]
We are working on a hotfix to address this.
